tests/test_parameter_mapping.py

```diff
@@ -1,7 +1,6 @@
+import numpy as np
 import pandas as pd
 import petab
```

Comment:
alphabetic ordering + own tools / petab last? very in favor.

Reply:
yeah
pycharm does the job for you (code -> optimize imports)